github/plugin.py

```diff
@@ -82,8 +82,23 @@ def __init__(self):
                 "user_data_required": True,
                 "help": "Delete a GitHub repository.",
                 "enabled": True     # It is HIGHLY recommended you have auth enabled for this!!
+            },
+            "!InviteUser": {
```

Comment:
I agree on removing it. After having it running with the two commands for a week now, I can see that this command is never used as the desired behaviour is actually the `AddUserToTeam` command.